pkg/models/search_user_filter.go

```diff
@@ -0,0 +1,103 @@
+package models
```

Comment:
I'm not sure this is the best location to put most of this code. I think a `filters` package in the `searchusers` service folder would be more appropriate (plus it would follow the same pattern as in the Enterprise code).

Reply:
You are right. I started to work with is from the beginning inside models and I didn't noticed about that it's better and simple replicate the Enterprise code.